web_interface/blueprints/composer.py

```
@@ -0,0 +1,901 @@
+                    'default': prop.get('default', ''),
+                    'description': prop.get('description', ''),
+                })
+        except Exception as e:
+            logger.warning("Failed to parse config_schema.json for %s: %s", plugin_id, e)
+
+    manifest = {}
+    if manifest_path.exists():
+        try:
+            manifest = json.loads(manifest_path.read_text())
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)